src/documentation/ToolkitDocumentation/CodeEmbed.tsx

```diff
@@ -115,35 +132,31 @@ interface CodePopUpProps extends BoxProps {
 // We draw the same code over the top in a portal so we can draw it
 // above the scrollbar.
 const CodePopUp = ({
-  setHovering,
   codeRef,
   concise,
   full,
+  background,
```

Comment:
> This looks a little daft on the bottom right hand corner. Specific example is in Explore -> variables -> Assigning values.
> 
> ![image](https://user-images.githubusercontent.com/95928279/149517993-c553b99d-5997-4e32-a597-9a9d4da999a4.png)
> 
> Should we set a min-width to avoid the insert code buttons over handing the edge of the code examples?

52b00df705ede7ab516c1a9e061b6ddd42695a73

I went for a fair bit longer than would have looked even with the drag handle in order to allow for more space in translations.

Reply:
52b00df705ede7ab516c1a9e061b6ddd42695a73